src/LineBlockTracer.h

```diff
@@ -33,6 +33,14 @@ class cLineBlockTracer :
 	typedef cBlockTracer super;
 
 public:
+	enum eLineOfSight
```

Comment:
This seems like it should be a bitset of flags rather than a set of three. What if a plugin wants to see through only water or only lava?